onnxscript/converter_test.py

```diff
@@ -559,6 +559,15 @@ def none_as_input(X):
         node = none_as_input.to_function_proto().node[0]
         self.assertEqual(node.input[1], "")
 
+    def test_unique_names_in_subscript_expr(self):
```

Comment:
`..._for_nested_index_expression`? since it looks like its testing this specific case?